src/main/java/cc/aabss/eventutils/UpdateChecker.java

```diff
@@ -35,36 +36,52 @@ private void notifyUpdate(@NotNull String latestVersion) {
     }
 
     public void checkUpdate() {
+        if (!EventUtils.MOD.config.updateChecker
+                || Versions.MC_VERSION == null
+                || Versions.EU_VERSION == null
+                || Versions.EU_VERSION_SEMANTIC == null) return;
+
+        // Get client
+        final MinecraftClient client = MinecraftClient.getInstance();
+        if (client.player == null) return;
```

Comment:
keep "Ensure client in-game" comment
we dont actually use the client for anything, so could probably just get rid of variable and make it:
```java
if (MinecraftClient.getInstance().player == null) return;
```